code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -77,6 +77,9 @@
 	if(alien && alien == IS_SKRELL)
 		M.adjustToxLoss(2 * removed)
 		return
+	else if(istype(M.species, /datum/species/xenochimera)) //vorestation addition
+		affect_ingest(M, alien, removed) //like hell are chimera of all things going to be poisoned by injecting them with meat
```

Comment:
Works for me. As far as I can tell the "injectable" var is literally only set for glucose. Why they didn't override glucose's affect_blood rather than checking a var for every nutriment type is beyond me.